var/spack/repos/builtin/packages/json-glib/package.py

```diff
@@ -22,6 +22,7 @@ class JsonGlib(MesonPackage):
     version('1.2.8', sha256='fd55a9037d39e7a10f0db64309f5f0265fa32ec962bf85066087b83a2807f40a', deprecated=True)
 
     depends_on('glib')
+    depends_on('gobject-introspection')
```

Comment:
Is this for all versions or a specific one on?

Reply:
honestly, I don't know, but it looks like all but one version is deprecated. Let me know if there is a change you would like me to do. I don't mind adding @1.6.6 or @1.6.6: if you think it is appropriate.